public/media/favicon/browserconfig.xml

```diff
@@ -3,6 +3,7 @@
 	<msapplication>
 		<tile>
 			<square70x70logo src="media/favicon/ms-icon-70x70.png"/>
+			<square144x144logo src="media/favicon/ms-icon-144x144.png"/>
```

Comment:
@petrKavulok Why did you add a browser config in demo app with absolute path with the slash at the beginning of the path https://github.com/TeskaLabs/asab-webui/blob/798db86905254f01f9ddcf88af8030f0689e474b/demo/public/media/favicon/browserconfig.xml#L5, but here you dont?

Reply:
honestly, I played along with the rest of config. I'm not sure how to test this